plugins/inputs/logql/client.go

```diff
@@ -0,0 +1,163 @@
+package logql
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	promcfg "github.com/prometheus/common/config"
+
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/internal"
+	common_http "github.com/influxdata/telegraf/plugins/common/http"
+)
+
+type client struct {
+	url      string
+	username config.Secret
+	password config.Secret
+	token    config.Secret
+	org      string
```

Comment:
The `org` field is defined but never populated from configuration. If this is intentional (reserved for future use), it's fine. Otherwise, it should be added to the `LogQL` struct and configuration.